### DISCUSSION TITLE
Inventor Control

### DISCUSSION THREAD

**Discussion by Davecl** [barban-dev/homeassistant-midea-dehumidifier#63] [Ideas]
Would it be possible for someone with more brain-power than me to adapt this to work with Inventor Control rather than Invmate?
My newer model won't connect to invmate as it has a different wifi hotspot that only works with inventor control.
i'm using the workround and connected via tuya but it is not ideal, very basic

**Comment by ando2040** [comment]
As far as I can tell, barban-dev 's integration won't work with later models/phone OS versions that require the Inventor Control app.
The dehumidifier CAN be added to Tuya Smart Life (instead of Inventor Control). Far from obvious so here we go:

In the Smart Life app... Add Device… Large Home Appliances… Air Conditioner (Wifi)
Then key in the wifi network you want it to connect to.
Then it will prompt to connect to the device’s access point (hold down Ion button)
Once connected to the Smartlife access point it will turn the device on, and the device will be added to the Smart Life app.
I just had to reload the Home Assistant Tuya integration I already had set up and it popped in. You could I believe add to LocalTuya by capturing the key from Tuya IOT site but I've not done that yet.
Disappointingly, the water level sensor does not seem to come in via the cloud route so I may have to mess with LocalTuya.
![image](https://github.com/user-attachments/assets/972f0e41-9899-4672-aa78-b405388b7f1f)
